src/ui/public/management/section.js

```diff
@@ -72,6 +72,11 @@ export default class ManagementSection {
    */
 
   getSection(id) {
-    return this.items.byId[id];
+    if (!id) return;
```

Comment:
Per the style guide, we should[ use braces for conditionals](https://github.com/elastic/kibana/blob/master/style_guides/js_style_guide.md#always-use-braces-for-conditionals-and-loops)